Source/SwiftLintFramework/Rules/FirebaseConfigDefaultsRule.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  FirebaseConfigDefaultsRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigDefaultsRule: ASTRule, RecursiveRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_defaults",
+        name: "Firebase Config Defaults",
+        description: "Firebase Config defaults should be set.",
+        nonTriggeringExamples: [
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "    remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+            "        (status, error) -> Void in \n" +
+            "    }\n" +
+            "    remoteConfig.setDefaults(fromPlist: \"RemoteConfigDefaults\")\n" +
+            "  }\n" +
+            "}",
+            "class ViewController: UIViewController {\n" +
+            "  override func viewDidLoad() {\n" +
+            "    super.viewDidLoad() \n" +
+            "    foo.fetch() { } \n" +
+            "    foo.fetch(fromURL: URL) { } \n" +
+            "  }\n" +
+            "}"
```

Comment:
Please consider adding these two examples:

```swift
            "class ViewController: UIViewController {\n" +
                "  override func viewDidLoad() {\n" +
                "    super.viewDidLoad()\n" +
                "  }\n" +
            "}",
            "class ViewController: UIViewController { }",
```

The first one will make sure an "empty" `viewDidLoad()` won't trigger a violation. The second one will make sure a View Controller that doesn't override `viewDidLoad()` won't trigger a violation as well.